src/components/deck-gl-wrapper/custom-tile-3d-layer.ts

```diff
@@ -0,0 +1,572 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-empty-function */
+import GL from "@luma.gl/constants";
```

Comment:
Could you wrap in comments new snipets of code in this file. Also It would be nice to have commented old code if you removed something.

Reply:
Done